effect_prn/models/end_of_study.py

```diff
@@ -25,40 +36,129 @@ class EndOfStudy(
         null=True,
     )
 
-    offschedule_reason = models.ForeignKey(
-        OffstudyReasons,
-        verbose_name="Reason patient was terminated from the study",
-        on_delete=models.PROTECT,
+    lastfollowup_datetime = models.DateTimeField(
```

Comment:
Unless there's a specific reason to use a `DateTimeField`, I'd go for a `DateField` here, and use `last_followup_date`